config/samples/config.yaml

```diff
@@ -1,26 +1,34 @@
-clusterName: "example-cluster"
+clusterName: "staging-usw2-k8s"
```

Comment:
This is very tricky to put the file in the samples directory, if it's only used for test cases, move it there or use a const string in the test cases.

Reply:
yeah, for tests let's collocate this type of thing with the test itself.

@whynowy what's your impression for what should go in `samples` directory? For one, I think everything should be an actual Kubernetes spec, which this isn't. Out of curiosity, would it be appropriate to put a Numaplane ConfigMap example in here (since it is part of Numaplane as opposed an object that is reconciled by Numaplane)?

Reply:
I think it would be good to put a config map example here for demonstration purpose.

Reply:
Specifically, you can put it under `tests/config/`.